### PR TITLE
chore(workflow-beta4): releasing workflow-beta4(-e2e)

### PR DESCRIPTION
**Pull request by jchauncey**

### CODE REVIEW

**Comment by bacongobbler** [comment]
chart LGTM. Time to test!

**Comment by jchauncey** [comment]
Wait we found an issue. `deisrel` defaulted the org in the params toml to `deis` instead of `deisci`

**Comment by bacongobbler** [comment]
ah good catch. I missed that

**Comment by jchauncey** [comment]
`putting all the images in one org++`

File: workflow-beta4/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+WARNING: this chart is for testing only! Features may not work, and there are likely to be bugs.
+
+Please report any issues you find in testing Workflow to the appropriate GitHub repository:
+- builder: https://github.com/deis/builder
+- chart: https://github.com/deis/charts
+- database: https://github.com/deis/postgres
+- helm classic: https://github.com/helm/helm-classic
+- logger: https://github.com/deis/logger
+- minio: https://github.com/deis/minio
```
Comment (mboersma) [inline workflow-beta4/README.md:13]: We should add links to deis/monitor, deis/fluentd, and deis/stdout-metrics here:

```
- fluentd: https://github.com/deis/fluentd
- monitor: https://github.com/deis/monitor
- stdout-metrics: https://github.com/deis/stdout-metrics
```

I'll make a PR to do the same against workflow-dev. _Edit_: see #262.

Comment (vdice) [inline workflow-beta4/README.md:13]: @mboersma see https://github.com/deis/charts/pull/261

Comment (jchauncey) [inline workflow-beta4/README.md:13]: done

**Pull request merged**